src/rgw/rgw_rados.cc

```diff
@@ -1505,6 +1506,7 @@ int RGWZoneParams::fix_pool_names()
   user_email_pool = fix_zone_pool_name(pool_names, name, ".rgw.users.email", user_email_pool.name);
   user_swift_pool = fix_zone_pool_name(pool_names, name, ".rgw.users.swift", user_swift_pool.name);
   user_uid_pool = fix_zone_pool_name(pool_names, name, ".rgw.users.uid", user_uid_pool.name);
+  roles_pool = fix_zone_pool_name(pool_names, name, ".rgw.authorizationdata", roles_pool.name);
```

Comment:
for multisite, we put some config objects (realm/zone/period) in the .rgw.root pool, and store sync logs in the log_pool. there are some 'list' commands to list config objects, but that stuff should keep working as long as we're calling `set_namespace()` consistently